toucan_connectors/hubspot_private_app/hubspot_connector.py

```diff
@@ -90,7 +90,7 @@ def _fetch_all(self, api: _Api) -> list[_HubSpotResult]:
         results = api.get_all()
         return [_HubSpotResult(**elem.to_dict()) for elem in results]
 
-    def _api_for_dataset(self, object_type: HubspotDataset) -> _Api:
+    def _api_for_dataset(self, object_type: HubspotDataset) -> _Api:  # pragma: no cover
```

Comment:
the pragma: no cover is really needed here ? since the function start with the _ underscore... ? just asking... i sonar yelling here ?

Reply:
Yup, we never execute it since it's mocked in the tests